exporters/otlp/otlpmetric/otlpmetricgrpc/client_test.go

```diff
@@ -329,3 +330,29 @@ func TestFailedMetricTransform(t *testing.T) {
 
 	assert.Error(t, exp.Export(ctx, testResource, otlpmetrictest.FailReader{}))
 }
+
+func TestNewExporterWithPartialSuccess(t *testing.T) {
+	mc := runMockCollectorWithConfig(t, &mockConfig{
+		partial: &collectormetricpb.ExportMetricsPartialSuccess{
+			RejectedDataPoints: 2,
+			ErrorMessage:       "partially successful",
+		},
+	})
+	defer func() {
+		_ = mc.stop()
+	}()
+
+	errors := otlpmetrictest.OTelErrors()
+
+	ctx := context.Background()
+	exp := newGRPCExporter(t, ctx, mc.endpoint)
+	require.NoError(t, exp.Export(ctx, testResource, oneRecord))
+
+	defer func() {
+		_ = exp.Shutdown(ctx)
+	}()
+
```

Comment:
```suggestion
	errors := otlpmetrictest.OTelErrors()
	ctx := context.Background()
	exp := newGRPCExporter(t, ctx, mc.endpoint)
	require.NoError(t, exp.Export(ctx, testResource, oneRecord))
	defer func() {
		_ = exp.Shutdown(ctx)
	}()
```
These blank lines are meaningless, right? Suggest removing them to keep it more readable.

Reply:
:shrug: I personally find blank lines improve readability.